src/pivot-table/hooks/use-column-width.ts

```diff
@@ -87,7 +87,7 @@ export default function useColumnWidth(
     [leftDimensionData.grid, getLeftColumnWidth],
   );
 
-  const rightGridWidth = useMemo(() => Math.max(rect.width - leftGridWidth - GRID_BORDER), [leftGridWidth, rect.width]);
+  const rightGridWidth = useMemo(() => rect.width - leftGridWidth - GRID_BORDER, [leftGridWidth, rect.width]);
```

Comment:
Why remove `Math.max`?

Reply:
because what it tries to evaluates is just a single value (`rect.width - leftGridWidth - GRID_BORDER`). There is no other value to compare and find the max between them